src/components/TableComponent.vue

```diff
@@ -38,6 +38,9 @@
                         :columns="columns"
                 ></table-row>
                 </tbody>
+                <tfoot>
+                    <slot name="tfoot" :rows="rows"></slot>
```

Comment:
I'm passing `rows` as it seems to be the most useful information, but please suggest changes if there's a better property.